### ISSUE TITLE
Electrum shows the "ProtocolError" exception some time after daemon started

### ISSUE THREAD

**Issue by MaxiMukh** [spesmilo/electrum#3107]
In my operating system **Ubuntu Linux - 14.04.1 LTS** with **Python 2.7.6**
Installed **Electrum - 2.9.2** and upgraded to **2.9.3** at last week
`{
    "auto_connect": true, 
    "blockchain_height": 491493, 
    "connected": true, 
    "fee_per_kb": 124088, 
    "path": "Here is the path to electrum", 
    "server": "elec.luggs.co", 
    "server_height": 491493, 
    "spv_nodes": 10, 
    "version": "2.9.3", 
    "wallets": {
        "Here is the path to wallet": true
    }
}`
When electrum daemon is running some time Electrum does not work correctly in command line. JSONRPC is still working and answering for requests, but it returns the same bitcoin payment address for all payment requests. Meanwhile for any Electrum command I'v got the following results:
(for ex: _**electrum version**_)
`File "/usr/local/bin/electrum", line 419, in <module>
    result = server.run_cmdline(config_options)
  File "/usr/local/lib/python2.7/dist-packages/jsonrpclib/jsonrpc.py", line 288, in __call__
    return self.__send(self.__name, args)
  File "/usr/local/lib/python2.7/dist-packages/jsonrpclib/jsonrpc.py", line 238, in _request
    check_for_errors(response)
  File "/usr/local/lib/python2.7/dist-packages/jsonrpclib/jsonrpc.py", line 567, in check_for_errors
    raise ProtocolError((code, message))
jsonrpclib.jsonrpc.ProtocolError: (-32603, u'Server error:   File "/usr/local/lib/python2.7/dist-packages/electrum/util.py", line 172, in print_stderr | IOError: [Errno 32] Broken pipe')`
(for ex: _**electrum daemon status**_)
`Traceback (most recent call last):
  File "/usr/local/bin/electrum", line 410, in <module>
    result = server.daemon(config_options)
  File "/usr/local/lib/python2.7/dist-packages/jsonrpclib/jsonrpc.py", line 288, in __call__
    return self.__send(self.__name, args)
  File "/usr/local/lib/python2.7/dist-packages/jsonrpclib/jsonrpc.py", line 238, in _request
    check_for_errors(response)
  File "/usr/local/lib/python2.7/dist-packages/jsonrpclib/jsonrpc.py", line 567, in check_for_errors
    raise ProtocolError((code, message))
jsonrpclib.jsonrpc.ProtocolError: (-32603, u'Server error:   File "/usr/local/lib/python2.7/dist-packages/electrum/util.py", line 172, in print_stderr | IOError: [Errno 32] Broken pipe')`
I have got the same problems after updating from 2.9.2 to 2.9.3 version. I found the similar Issue which has been resolved by periodically-running script with "electrum daemon stop && electrum daemon start". Unfortunately that solution does not work for me, just server rebooting.
**Please, let me know what the problem is and how I can solve it?**

**Comment by wakiyamap** [comment]
fmm....
I ran electrum(2.9.3) on python2.7.12 .
May you try upgrading Python2 .

**Comment by MaxiMukh** [comment]
In operating system currently installed:
```
python2.7 - 2.7.6-8ubuntu0.3 - amd64
```
with many other python modules.
When I try update packages from repository:
**apt-get install python2.7**
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
python2.7 is already the newest version.
python2.7 set to manually installed.
0 upgraded, 0 newly installed, 0 to remove and 194 not upgraded.
```
I believe that repository contain stable versions.

**Comment by ecdsa** [comment]
try with 3.0

**Comment by MaxiMukh** [comment]
@ecdsa 
Electrum client 3.0 version is not in a release status.
You are have not instructions for installing 3.0 version on site:
https://www.electrum.org/#download
Why you recommended try 3.0 version as solving the problem ?
Is there any way to solve exactly my problem ?

**Comment by ecdsa** [comment]
no, I recommend to wait.
but I will not investigate your issue because it is with 2.9.2

**Comment by MaxiMukh** [comment]
@ecdsa 
No, It is not only in 2.9.2 version. I wrote about that:
"_Installed Electrum - 2.9.2 and upgraded to 2.9.3 at last week_"
It was began in 2.9.2 version, but in 2.9.3 version it still affected too.
We discussed about that in our team and came to a conclusion,
so that "ProtocolError" exception began showing after 
our colleague imported the wallet into mobile application, using XPUB.
May be it was coincidence, but maybe it will be a small clue.
